### PR TITLE
Add FAQ section w/ questions

### LINKED ISSUE #25: FAQ Section
* Two-column layout
* Fully visible questions & answers, no accordion
**Inspiration:**
  * https://www.hoohacks.io

### PR DESCRIPTION
**Pull request by samarthdave**
Closes #25 

- clean up index.ejs
- convert all (most?) css files to 4 spaces
- create faq partial

### CODE REVIEW

**Comment by samarthdave** [comment]
Won't be closed yet since we'll keep adding to FAQ section and styling isn't done. Are we going to stick with 4 spaces? Some files have 2 and others have 4 ...

**Comment by phulsechinmay** [comment]
Can you change the background color and add the transitions that the design specifies? I've already added them in the meet-the-team branch, so you can copy the HTML / CSS from there if needed.

**Comment by samarthdave** [comment]
what should we do about anchor font color with the green background? I can try white with underline or dark blue. @phulsechinmay 
image

**Comment by phulsechinmay** [comment]
I went ahead and added the background and transitions. The section might need padding / margin

File: scss/custom.scss [hunk 1]
```diff
@@ -76,39 +76,37 @@ h6 {
...
- .slick-next:before {
-   color: black;
- }
+.slick-slide img {
+    width: 100%;
+}
 
+.slick-prev:before,
+.slick-next:before {
+    color: black;
```
Comment (phulsechinmay) [inline scss/custom.scss:98]: We might need to create a .eslintrc file to lay down format standards, will help reduce diffs like these @geooot

File: views/partials/index/faq.ejs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            <div class="col-md">
+                <h1 class="section-title">
+                    Frequently Asked Questions
+                </h1>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md">
+                <div class="question-block">
+                    <h5 class="question">What is TAMU Datathon?</h5>
```
Comment (phulsechinmay) [inline views/partials/index/faq.ejs:15]: All headers are Roboto Mono, do we wanna change the headers in FAQs to Work Sans for consistency in the section?

**Comment by geooot** [comment]
could we try a pastel green color (like the about section) to see how it looks?

**Comment by geooot** [comment]
I just made some changes to fix the section padding, and matched the font-size with the other sections.

Attached is a screenshot of how a more pastel color could work. I think it makes the FAQ section more readable.
image

**Pull request merged**